code/modules/antagonists/changeling/datum_changeling.dm

```diff
@@ -162,7 +162,7 @@
 		if(prob(70))
 			add_objective(/datum/objective/escape)
 		else
-			add_objective(/datum/objective/escape/escape_with_identity)
+			add_objective(/datum/objective/escape/escape_with_identity) // If our kill target has no genes, 30% time pick someone else to steal the identity of
```

Comment:
This comment isn't really true. It only gives this objective as a way of adding variety instead of choosing escape all the time.

Reply:
The prob(70) check will only run if no escape objective is added by the earlier kill objective, which only happens if the kill target is geneless. 

I could probably remove the comment but I added when I was confused looking at this code, hopefully to make it easier for other contributors

Reply:
Yeah nevermind, this is fine.